macro/run_trac_ca_its.C

```diff
@@ -54,7 +53,7 @@ void run_trac_ca_its(bool useITSVertex = false,
   gSystem->Load("libO2ITStracking.so");
 
   std::unique_ptr<GPUReconstruction> rec(GPUReconstruction::CreateInstance());
```

Comment:
@mconcas : This will be how you can get the ITS traits. Just use GetITSVertexerTraits instead of GetITSTrackerTraits

Reply:
Thx, that was quite straightforward, probably, since the CI is still stuck I might think to update https://github.com/AliceO2Group/AliceO2/pull/2194/files.
Still not able to load the CUDA traits despite forcing architecture

Reply:
It will only work properly once this PR is merged (after I fixed my stupid bug that failed in the CI...) Anyhow, CUDA is not yet working, also for me. Don't understand why yet, but I would not count on the traits for CUDA working yet either.